test/client-tests.js

```diff
@@ -326,14 +326,16 @@ describe('NamedCacheClient IT Test Suite', function () {
     describe('truncate()', () => {
       it('should clear the cache without raising listener events', async () => {
         let deleteCount = 0
-        cache.addMapListener({
-          entryDeleted (event) { deleteCount++ }
-          , entryInserted (event) {
-          }, entryUpdated (event) {
+        const listener = new (class MyListener extends event.MapListener {
+          constructor () {
+            super()
+            this.on(event.MapEventType.DELETE, (event) => {
+              deleteCount++
+            })
           }
         })
```

Comment:
This could be rewritten as:
```javascript
const listener = new event.MapListener()

listener.on(event.MapEventType.DELETE, (event) => { deleteCount++ })
```
Much cleaner than a dummy class, constructor override, etc.

Reply:
It would be nice to change imports as well, so you don't have to do `event.Xyz` all the time, too.